md/database-configuration.md

```diff
@@ -244,10 +244,9 @@ First make sure that you have [configured Business Data](database-configuration-
 
 Now that you are almost done with the switch from h2 to your chosen RDBMS, you can remove h2:
 
-* For JBoss
-   * Delete the `$JBOSS_HOME/standalone/deployments/h2.sar` folder.
-   * Remove the configuration for h2 from `<JBOSS_HOME>/standalone/configuration/standalone.xml`.
-   * Make sure that `sysprop.bonita.db.vendor` property in `<JBOSS_HOME>/standalone/configuration/standalone.xml` is not set to h2\.
+* For Wildfly
+   * Remove the configuration for h2 from `<WILDFLY_HOME>/standalone/configuration/standalone.xml`.
+   * Make sure that `sysprop.bonita.db.vendor` property in `<WILDFLY_HOME>/standalone/configuration/standalone.xml` is not set to h2\.
 * For Tomcat
    * Remove h2 jar files (`bonita-jboss-h2-mbean-1.0.0.jar`, `bonita-tomcat-h2-listener-1.0.1.jar`, `h2-1.3.170.jar`). Files are located in: `<TOMCAT_HOME>/lib` or in `<TOMCAT_HOME>/lib/bonita`.
```

Comment:
Shouldn't `bonita-jboss-h2-mbean-1.0.0.jar`, `bonita-tomcat-h2-listener-1.0.1.jar` already be removed from default installatio, with AUTOSERVER mode?